### DISCUSSION TITLE
It doesn't copy any files

### DISCUSSION THREAD

**Discussion by thany** [sapphi-red/vite-plugin-static-copy#114] [Q&A] [answered]
In the log it says:

```
[vite-plugin-static-copy] Collected 5 items.
```

So I'm like "great..... and where are they?" because I then go to the specified target directory, and they're not there. So the question becomes two-fold in my mind:

1. Where are the files copied to, if not the directory specified?
2. How to enable verbose logging, e.g. to see which file is being copied where?

This greatly helps in setting things up, as opposed to "blindly" having to trust that the plugin and I both have the same idea of what both directories are relative to, and how they are expected to be formatted.

Here's my config:

```js
    viteStaticCopy({
      targets: [
        {
          src: 'public/assets/images/*',
          dest: 'assets/Platform3/images',
        },
      ],
    }),
```

The `outDir` is set to `'dist'`. So while it has collected 5 items (or claims to have), the dist folder remains empty. I've checked, and the target folder definitely has the correct permissions set. So is `dest` not formatted correctly or something? If only it would verbosely show what's going on.

**Comment by sapphi-red** [comment]
This plugin does not copy files during dev. They are served directly from dev server.
https://github.com/sapphi-red/vite-plugin-static-copy#differences-with-rollup-plugin-copy

**Reply by thany** [reply]
Well then you know what the next question is. What's the answer? 😀

**Reply by sapphi-red** [reply]
I don't know what your next question is.

**Reply by thany** [reply]
I thought it was obvious. It's the exact polar opposite of your explanation: how to make it copy files during dev?

**Reply by sapphi-red** [reply, marked as answer]
That is currently out of scope of this plugin (see https://github.com/sapphi-red/vite-plugin-static-copy/issues/36), so you can't do it with this plugin.
